doc/tutorial/climada_entity_Exposures.ipynb

```diff
@@ -28,46 +28,57 @@
+    "A \"geometry\" column is initialized in the `GeoDataFrame` of the `Exposures` object, other columns are optional at first but some have to be present or make a difference when it comes to do calculations.\n",
+    "Apart from these special columns the data frame may contain additional columns, they will simply be ignored in the context of CLIMADA.\n",
+    "\n",
+    "The full list of meaningful columns is this:"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "\n",
-    "<br>\n",
-    "\n",
-    "| Mandatory columns | Data Type | Description |\n",
-    "| :-------------------- | :------------ | :------------------------------------------------------------------------------------- |\n",
-    "| `latitude` | float | latitude |\n",
-    "| `longitude` | float | longitude |\n",
-    "| `value` | float | a value for each exposure &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|\n",
+    "| Column | Data Type | Description | Meaningful in |\n",
+    "| :-------------------- | :------------ | :------------------------------------------------------------------------------------- | - |\n",
+    "| `geometry` | Point | the geometry column of the `GeoDataFrame`, i.e., latitude (y) and longitude (x) | centroids assignment |\n",
+    "| `value` | float | a value for each exposure | impact calculation |\n",
+    "| `impf_*`  | int | impact functions ids for hazard types.<br>important attribute, since it relates the exposures to the hazard by specifying the impf_act functions.<br>Ideally it should be set to the specific hazard (e.g. `impf_TC`) so that different hazards can be set<br>in the same Exposures (e.g. `impf_TC` and `impf_FL`). | impact calculation |\n",
+    "| `centr_*` | int | centroids index for hazard type.<br>There might be different hazards defined: centr_TC, centr_FL, ...<br>Computed in method `assign_centroids()` | impact calculation |\n",
+    "| `deductible` | float | deductible value for each exposure. <br>Used for insurance | impact calculation |\n",
+    "| `cover` | float | cover value for each exposure. <br>Used for insurance | impact calculation |\n",
+    "| `region_id` | int | region id (e.g. country ISO code) for each exposure | aggregation |\n",
+    "| `category_id` | int | category id (e.g. building code) for each exposure | aggregation |"
```

Comment:
Are all these columns required now? If not, maybe you could detail which one are optional.